src/components/Button/styles.css

```diff
@@ -0,0 +1,119 @@
+:root {
```

Comment:
Is there a good way to grok CSS in code review apart from syntax/typos and a bit of logic? It feels like the kind of thing that does best with acceptance testing within the browser itself?

Reply:
Yeah, good q. We don't deploy a preview branch for this, which would be the usual way to debug / inspect the styles here and make sure everything visually looked ok.